app/controllers/v0/sessions_controller.rb

```diff
@@ -65,14 +66,14 @@ def parse_date(date_string)
     # Ruby-Saml does not parse the <samlp:Response> xml so we do it ourselves to find
     # which LOA was performed on the ID.me side.
     # TODO - remove this method once LOA is returned as a SAML Attribute
-    def level_of_assurance
+    def parse_current_loa
       raw_loa = Hash.from_xml(@saml_response.response)
                     .dig('Response', 'Assertion', 'AuthnStatement', 'AuthnContext', 'AuthnContextClassRef')
       LOA::MAPPING[raw_loa.to_sym]
     end
 
     def create_new_user
-      if user_attributes[:level_of_assurance] == LOA::ONE
+      if user_attributes[:loa_current] == LOA::ONE
```

Comment:
Hm, it turns out this code returns `nil` if user.loa1?==true.  Changed ti this instead:

```ruby
user = User.new(user_attributes)
user = Decorators::MviUserDecorator.new(user).create unless user.loa1?
user
```